### PR TITLE
Add some icons to the toolbar

### PR DESCRIPTION
**Pull request by mee4895**

### CODE REVIEW

File: ICGE-Ui/src/main/java/de/unistuttgart/informatik/fius/icge/ui/internal/SwingTextureStorage.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import de.unistuttgart.informatik.fius.icge.ui.TextureRegistry;
+
+/**
+ * The SwingTextureStorage contains all icon ids needed by swing ui components
+ *
+ * @author Tobias Wältken
+ * @version 1.0
+ */
+public abstract class SwingTextureStorage {
```
Comment (buehlefs) [inline ICGE-Ui/src/main/java/de/unistuttgart/informatik/fius/icge/ui/internal/SwingTextureStorage.java]: Maybe rename to `StaticUiTextures` (and rename init to load)

File: ICGE-Ui/src/main/java/de/unistuttgart/informatik/fius/icge/ui/internal/SwingTextureStorage.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+     *
+     * @param registry The registry to load the icons in
+     */
+    public static void init(TextureRegistry registry) {
+        SwingTextureStorage.playIcon = registry.loadTextureFromResource(
+            "textures/play.png",
+            TextureRegistry.class::getResourceAsStream
+        );
+        SwingTextureStorage.pauseIcon = registry.loadTextureFromResource(
+            "textures/pause.png",
```
Comment (buehlefs) [inline ICGE-Ui/src/main/java/de/unistuttgart/informatik/fius/icge/ui/internal/SwingTextureStorage.java]: Make pasue icon more visible in the UI (use an darker/orange color).

Comment (mee4895) [reply ICGE-Ui/src/main/java/de/unistuttgart/informatik/fius/icge/ui/internal/SwingTextureStorage.java]: This icon wasn't uploaded by me don't know where it came from but yes I can do the recoloring.

Comment (buehlefs) [reply ICGE-Ui/src/main/java/de/unistuttgart/informatik/fius/icge/ui/internal/SwingTextureStorage.java]: It was a rush-job by me for the ICGE (v1). We had no time to fix it then but now we have the time...

**Pull request merged**